init.sql

```diff
@@ -159,3 +159,202 @@ create table asset_creators
 create unique index asset_creators_asset_id on asset_creators (asset_id);
 create index asset_creator on asset_creators (asset_id, creator);
 create index asset_verified_creator on asset_creators (asset_id, verified);
+
+create type whitelist_mint_mode AS ENUM ('burn_every_time', 'never_burn');
+create type end_setting_type AS ENUM ('date', 'amount');
+
+create table candy_machine
+(
+    id                       bytea               PRIMARY KEY,
+    features                 int,
+    authority                bytea               not null,
+    mint_authority           bytea,
+    wallet                   bytea               not null,
+    token_mint               bytea,
+    items_redeemed           int                 not null,
+    candy_guard_pda          bytea,
+    version                  int                 not null,
+    collection_mint          bytea,                            
+    allow_thaw               bool,                                    
+    frozen_count             int,                                      
+    mint_start               int,
+    freeze_time              int,                                     
+    freeze_fee               int,                                      
+);
+
+create table candy_machine_data
```

Comment:
Overall looks like you have merged the data for CMV2 and CMV3/Candy Core.  One other option is that we say we just don't index CMV2.  Have we talked about the advantages/disadvantages of that?

Reply:
this was original request from @austbot first pr review to combine the stuff from core and non core cm. but could have been more than intended.